test/mock/kernel/mock_framework.zig

```diff
@@ -377,11 +393,11 @@ fn Mock() type {
         pub fn addAction(self: *Self, comptime fun_name: []const u8, data: var, action_type: ActionType) void {
             // Add a new mapping if one doesn't exist.
             if (!self.named_actions.contains(fun_name)) {
-                expect(self.named_actions.put(fun_name, TailQueue(Action).init()) catch unreachable == null);
```

Comment:
I had to change this because `put` now returns void so there's nothing to compare with `null`.